pkg/globalnet/controllers/types.go

```diff
@@ -123,10 +123,11 @@ type globalEgressIPController struct {
 }
 
 type egressPodWatcher struct {
-	stopCh      chan struct{}
-	ipSetName   string
-	namedIPSet  ipset.Named
-	podSelector *metav1.LabelSelector
+	stopCh       chan struct{}
+	ipSetName    string
+	namedIPSet   ipset.Named
+	podSelector  *metav1.LabelSelector
+	AllocatedIPs []string
```

Comment:
nit - for consistency.

```suggestion
	allocatedIPs []string
```